src/Sulu/Bundle/AdminBundle/Command/UpdateBuildCommand.php

```diff
@@ -101,7 +101,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
             'index.js',
             'package.json',
             'webpack.config.js',
-            '.babelrc',
+            'babel.config.json',
```

Comment:
@nnatter I think we need to rename `.babelrc` if it exists to `babel.config.json`